pages/common/protoc.md

```diff
@@ -0,0 +1,19 @@
+# protoc
+
+> Parse Google Protobuf `.proto` files and generate output in the specified language.
+
+- Generate Python code from a `.proto` file:
+
+`protoc --python_out={{output_directory}} {{input_file}}`
+
+- Generate Java code from a `.proto` file that imports other `.proto` files:
+
+`protoc --java_out={{output_directory}} --proto_path={{import_search_path}} {{input_file}}`
+
+- Generate Java code from a `.proto` file that imports other `.proto` files:
```

Comment:
Repeated. We can remove this.

Reply:
Whoops, no idea how that happened :) 
fixed